src/django_charting/__init__.py

```diff
@@ -0,0 +1,19 @@
+from django_charting.charts import Chart
+from django_charting.columns import BaseColumn, NumberColumn, StringColumn
+
+try:
+    from importlib.metadata import metadata
+except ImportError:
+    from importlib_metadata import metadata
+
+meta = metadata("django-charting")
+
+__version__ = meta["Version"]
+
+__all__ = [
+    "__version__",
+    "Chart",
+    "BaseColumn",
+    "StringColumn",
+    "NumberColumn",
+]
```

Comment:
Wellicht missen er nog meer, maar deze viel me op:

```suggestion
    "DateColumn",
]
```

Reply:
Fixed